Tests/MongoSwiftSyncTests/MongoCollectionTests.swift

```diff
@@ -540,13 +540,28 @@ final class MongoCollectionTests: MongoSwiftTestCase {
         expect(result3).to(equal(["_id": 2, "cat": "blah"]))
         expect(try self.coll.countDocuments()).to(equal(2))
 
-        // test invalid maxTimeMS throws error
+        // test using a hint
```

Comment:
since hint is only supported in MongoDB 4.4+, we'll need to skip this test on earlier versions to ensure the tests pass. Because of this, it probably makes sense to break the hint stuff out to its own test case too.